sources/packages/web/src/components/aest/students/EditCASSupplierInfoModal.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <v-form ref="addCASSupplierForm">
+    <modal-dialog-base
+      title="Update supplier/Site information"
+      :showDialog="showDialog"
+    >
+      <template #content>
+        <error-summary :errors="addCASSupplierForm.errors" />
+        <div class="pb-5">
+          <span class="label-value"
+            >Enter new supplier number and/or site number for this student to
+            maintain consistency with CAS validations.</span
```

Comment:
The ticket AC states both fields are mandatory, hence the `and/or` in the sentence does not make sense.